docs/includes/v3_backup_steps_common.rst

```diff
@@ -0,0 +1,146 @@
+.. note::  The instructions below assume that you are using the same *Admin Workstation*
+   that was used to manage your old instance. If you are using a new *Admin
+   Workstation* you will need to copy the directory ``~amnesia/Persistent/securedrop``
+   from the old workstation to the new workstation (using a *Transfer Device*)
+   before proceeding.
+
+#. If you have not already done so, :ref:`back up the existing installation <backing_up>`. The instructions below assume that the backup has been created and renamed ``sd-backup-old.tar.gz``.
```

Comment:
I would just add this as a step, "Rename the backup file to .."

Reply:
This one I think I'm going to leave (this naming is part of @zenmonkeykstop's patch), because elsewhere it refers to assuming filenames like `ossec.asc`, `journalist.asc`, `ca.crt`, etc, so I think it's clear that those are placeholder names.

Reply:
This is the only change I didn't incorporate (a step called "Rename the backup file to..."). Rationale: there are instructions throughout that reference 'placeholder' filenames (`ossec.asc`, `journalist.asc`, `ca.crt`, etc), and I think if we say "we assume the file is called X" it's clear enough.